string shape_name
---
geometry_msgs/Pose object_pose